getting-started/vs-code-integration/convert-project-wizard.md

```diff
@@ -0,0 +1,85 @@
+---
+title: Convert to Telerik Project
+page_title: Convert to Telerik Project - Visual Studio Code Integration
+description: Learn how to convert your existing Blazor app to use Telerik components in it with a few clicks through our Visual Studio Code Templates.
+slug: getting-started-vs-code-integration-convert-project
+position: 3
+---
+
+# Convert to Telerik Application
+
+This article demonstrates how to enable the Progress&reg; Telerik&reg; UI for Blazor components in your existing Blazor app with a few mouse clicks.
```

Comment:
Add a few words to denote that this is about the VS Code environment. Also, this is not mouse clicks, but with a simple VS Code command. 

Perhaps the whole sentence can read something like "This article demonstrates how to enable the Progress&reg; Telerik&reg; UI for Blazor components in your existing Blazor app with a single VS Code command."